cmd/agent/api/agent/agent.go

```diff
@@ -40,7 +40,7 @@ import (
 func SetupHandlers(r *mux.Router) *mux.Router {
 	r.HandleFunc("/version", common.GetVersion).Methods("GET")
 	r.HandleFunc("/hostname", getHostname).Methods("GET")
-	r.HandleFunc("/flare", makeFlare).Methods("POST")
+	r.HandleFunc("/flare", makeFlare).Methods("POST").Queries("profileDir", "{path:.*}")
```

Comment:
I am wondering if it coud lead to a security issue as this code returns a copy of the files in `profileDir` with the access rights of the running agent.

Reply:
Technically it just returns the path to the temporary files in `profileDir`, I'm not sure if that's a security issue 🤔